deploy/crds/aws.managed.openshift.io_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml

```diff
@@ -82,6 +83,11 @@ spec:
           - "route53resolver:GetResolverRuleAssociation"
           - "route53resolver:ListResolverRuleAssociations"
           - "route53resolver:ListResolverRules"
+          - "route53:CreateVPCAssociationAuthorization"
+          - "route53:DeleteVPCAssociationAuthorization"
+          - "route53:ListVPCAssociationAuthorizations"
```

Comment:
these shouldn't be required. these permissions are only required in the PrivateLink account where the hosted zone exists.

i think we need
```
route53:AssociateVPCWithHostedZone
route53:DisassociateVPCFromHostedZone

ec2:DescribeVpcs
```

the last one coming from AWS requirement.. https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/r53-api-permissions-ref.html#required-permissions-private-hosted-zones

Reply:
@abhinavdahiya - The reason I added these was due to the Note here for AWS's API documentation: https://docs.aws.amazon.com/Route53/latest/APIReference/API_AssociateVPCWithHostedZone.html

I can remove for now if you feel we can move forward without them.

Reply:
I've removed those for now, hopefully we will not need them.